.github/workflows/tests.yml

```diff
@@ -7,17 +7,13 @@ on:
 env:
   DRIVER_URL: "http://localhost:4444/wd/hub"
 
-defaults:
-  run:
-    shell: bash
-
 jobs:
-
   tests:
     name: Tests
     runs-on: ubuntu-20.04
     strategy:
       matrix:
+        selenium: [ '2.53.1', '3.141.59' ]
```

Comment:
I suggest sparing resources by having a single job changing the selenium version instead of doing that for all PHP versions.